README.md

```diff
@@ -1,2 +1,15 @@
 # auth-server
 Practicing Authentication Strategies
+<!-- 
+
```

Comment:
Here is the link to the [README template](https://github.com/codefellows/seattle-javascript-401n17/blob/master/reference/submission-instructions/labs/README-template.md) if you need it.